dstep/main.d

```diff
@@ -63,7 +63,9 @@ auto parseCLI (string[] args)
         "no-comments", "Disable translation of comments.", &config.noComments,
         "public-submodules", "Use public imports for submodules.", &config.publicSubmodules,
         "package", "Specify package name.", &config.packageName,
-        "dont-reduce-aliases", "Disable reduction of primitive type aliases.", &config.dontReduceAliases);
+        "dont-reduce-aliases", "Disable reduction of primitive type aliases.", &config.dontReduceAliases,
+        "single-line-function-headers", "Do not break function headers to multiple lines.", &config.singleLineFunctionHeaders,
+        "no-space-after-function-name", "Do not put a space after a function name.", &config.noSpaceAfterFunctionName);
```

Comment:
I rethought the matter and `getopt` seems to support assigning of a `true` and `false` to the CLI switch. Maybe it would be better to resign from `no-` prefixed switches and require users to specify the desirable value explicitly instead. E.g.

`--reduce-aliases=false`
`--comments=true`

I just want to point out that possibility.